assets/queries/cloudFormation/docdb_cluster_master_user_password_rule/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "DocDB_Cluster_Master_User_Password_Rule",
+  "queryName": "DocDB Cluster Master User Password Rule",
```

Comment:
Change **id** and **query directory** name to: `docdb_cluster_master_password_in_plaintext`
Change **query nam**e to: `DocDB cluster master password in plaintext`